style.css

```diff
@@ -157,4 +157,82 @@ div {
         background-color: darkslategrey;
         line-height: 760px;
     }
+}
+
+@media only screen and (min-width: 2560px) { 
```

Comment:
i think you meant to do min-width to 256px and for a tablet view you need a max width as well. this line should be @media only screen and (min-width: 256px) and (max-width: 758px)